src/app/core/entity-components/entity-form/entity-form.service.ts

```diff
@@ -37,15 +37,14 @@ export class EntityFormService {
     formField.view =
       formField.view ||
       this.entitySchemaService.getComponent(propertySchema, "view");
+    formField.tooltip = formField.tooltip || propertySchema?.description;
     if (forTable) {
       formField.forTable = true;
-      formField.placeholder =
-        formField.placeholder ||
-        propertySchema.labelShort ||
-        propertySchema.label;
+      formField.label =
+        formField.label || propertySchema.labelShort || propertySchema.label;
     } else {
       formField.forTable = false;
-      formField.placeholder = formField.placeholder || propertySchema.label;
+      formField.label = formField.label || propertySchema.label;
```

Comment:
In case of the above `formField.tooltip || propertySchema?.description` it is possible that a form field neither has a `tooltip` nor a `propertySchema`. In case of the label one of the both HAS to be available. Which means if `formField.label` is undefined, then the `propertySchema` needs to be defined and has to provide the label. If `propertySchema` is undefined, then an error should be thrown.